tests/unit/PostgrestFilterTest.php

```diff
@@ -27,118 +29,261 @@ public function setup(): void
     public function testEq(): void
     {
         $result = $this->filter->eq('name', 'Algeria');
```

Comment:
printing $result will give you the whole object that Is being sent to the Request.